Broadcast analysis options to child experiments (#1277)
Qiskit-Extensions/qiskit-experiments: docs/tutorials/getting_started.rst

    parallel_data = parallel_exp.run(backend, seed_simulator=101).block_for_results()

    for result in parallel_data.analysis_results():
        print(result)

Broadcating analysis option to child experiments
------------------------------------------------

> If we would like to set analysis options to each of the child experiments, we can use the keyword
> 'broadcast' when configuring an analysis option for a composite experiment.

Reviewer: ```suggestion
Use the `broadcast` parameter to set analysis options to each of the child experiments.
```

.. jupyter-execute::

    parallel_exp.analysis.set_options(plot=False, broadcast=True)

> When the 'broadcast' flag is set, the specified option will be applied to each child experiment.

Reviewer: ```suggestion
When the `broadcast` flag is set to `True`, the specified option will be applied to each child experiment.
```

> In this instance, the analysis will not generate a figure for the child experiment after the analysis.

Reviewer: It is not clear from the explanation whether this works also recursively to nested experiments.

Author: If the type of the nested experiments' class is `CompositeExperiment` it will work on them to. Should I add explanation like:
```
When the 'broadcast' flag is set to `True`, the specified option will be applied to each child experiment.
If the child experiment is a complex experiment that inherits from :class:`.CompositeExperiment` (such as :class:`.ParallelExperiment` and :class:`.BatchExperiment` classes), this process will work recursively.
```

Reviewer: Yes, I'd omit a few words:
```
If the child experiment inherits from :class:`.CompositeExperiment` (such as :class:`.ParallelExperiment` and :class:`.BatchExperiment` classes), this process will continue to work recursively.
```

Author: Changed as suggested.
